config/7.1.0/securebanking/ig/scripts/groovy/GetResourceOwnerIdFromConsent.groovy

```diff
@@ -180,7 +180,12 @@ if (request.getMethod() == "GET" || request.getMethod() == "POST") {
                     return newResultPromise(getErrorResponse())
                 }
 
-                attributes.put("amount", intentResponseObject.OBIntentObject.Data.Initiation.InstructedAmount.Amount)
+                if (intentResponseObject.OBIntentObject.Data.Initiation.InstructedAmount != null && intentResponseObject.OBIntentObject.Data.Initiation.InstructedAmount.Amount != null) {
```

Comment:
I think that will more safe use the `intentType` to know which field we need to use to retrieve the amount.
- Check line 124 where we get the intentType.
Suggested solution:
```java
if(intentType.equals(IntentType. DOMESTIC_VRP_PAYMENT_CONSENT) {
    attributes.put("amount", intentResponseObject.OBIntentObject.Data.ControlParameters.MaximumIndividualAmount.Amount)
} else {
    attributes.put("amount", intentResponseObject.OBIntentObject.Data.Initiation.InstructedAmount.Amount)
}